coordinator/src/main/java/com/tencent/rss/coordinator/CoordinatorMetrics.java

```diff
@@ -30,11 +30,17 @@
   private static final String RUNNING_APP_NUM = "running_app_num";
   private static final String TOTAL_APP_NUM = "total_app_num";
   private static final String EXCLUDE_SERVER_NUM = "exclude_server_num";
+  private static final String TOTAL_ACCESS_REQUEST = "total_access_request";
+  private static final String TOTAL_CANDIDATES_DENIED_REQUEST = "total_candidates_denied_request";
+  private static final String TOTAL_LOAD_DENIED_REQUEST = "total_load_denied_request";
 
   static Gauge gaugeTotalServerNum;
   static Gauge gaugeExcludeServerNum;
   static Gauge gaugeRunningAppNum;
   static Counter counterTotalAppNum;
+  static Counter counterTotalAccessRequest;
```

Comment:
It should be included in grpc metrics

Reply:
As discussed offline, these are business logic counters, so put them in the CoordinatorMetrics